temba/channels/views.py

```diff
@@ -1844,6 +1846,59 @@ def form_valid(self, form):
 
             return HttpResponseRedirect(reverse('channels.channel_configuration', args=[channel.id]))
 
+    class ClaimLine(OrgPermsMixin, SmartFormView):
+        class LineForm(forms.Form):
+            channel_id = forms.CharField(label=_("Channel ID"), required=True, help_text=_("The ID of the your Channel on Business LINE"))
+            channel_secret = forms.CharField(label=_("Channel Secret"), required=True, help_text=_("The Secret of the your Channel on Business LINE"))
+            channel_mid = forms.CharField(label=_("Channel MID"), required=True, help_text=_("The MID of the your Channel on Business LINE"))
```

Comment:
Yes, they call "Channels" at LINE developers console. But, also has reference with "BOT API", so, could be "The MID of the Line bot"?